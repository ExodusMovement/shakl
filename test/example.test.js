import React from 'react';

import { create as r } from 'react-test-renderer';

import Example from './Example';

it('works', () => {
  const app = r(<Example />).toJSON();

  expect(app).toMatchSnapshot();
});
