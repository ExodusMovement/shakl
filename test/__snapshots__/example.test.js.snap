// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
<RCTScrollView
  backgroundColor="#f5fcff"
  contentContainerStyle={
    {
      "flex": 1,
      "paddingTop": 20,
    }
  }
  style={
    {
      "backgroundColor": "#f5fcff",
    }
  }
>
  <View>
    <View
      style={
        {
          "backgroundColor": "red",
          "height": 40,
          "width": 40,
        }
      }
    />
    <Text
      style={
        {
          "fontSize": 28,
        }
      }
    >
      Hello world
    </Text>
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "backgroundColor": "green",
          "height": 40,
          "opacity": 1,
          "width": 40,
        }
      }
    />
    <View
      style={
        {
          "backgroundColor": "blue",
          "height": 40,
          "width": 40,
        }
      }
    />
    <Text
      style={
        {
          "backgroundColor": "red",
          "padding": 0,
        }
      }
    >
      Hello world
    </Text>
    <Text
      padded={true}
      style={
        {
          "backgroundColor": "red",
          "padding": 10,
        }
      }
    >
      Hello world
    </Text>
    <Text
      style={
        {
          "fontSize": 20,
        }
      }
    >
      Hello world
    </Text>
    <Text
      style={
        {
          "fontSize": 20,
          "fontWeight": "bold",
        }
      }
    >
      Hello world
    </Text>
    <Text
      style={
        {
          "color": "green",
          "fontSize": 20,
          "fontWeight": "bold",
        }
      }
    >
      Hello world
    </Text>
    <Text
      numberOfLines={1}
      style={
        {
          "color": "blue",
        }
      }
    >
      Hello world Hello world Hello world Hello world Hello world 
    </Text>
    <View
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "backgroundColor": "green",
          "height": 40,
          "width": 40,
        }
      }
    >
      <View
        style={
          {
            "backgroundColor": "red",
            "height": 40,
            "width": 40,
          }
        }
      />
    </View>
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "flex": 1,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          {
            "color": "green",
          }
        }
      >
        BUTTON
      </Text>
    </View>
  </View>
</RCTScrollView>
`;
